xml/admin-neutron-config-az.xml

```diff
@@ -292,9 +292,6 @@ $ openstack network agent show 9632309a-2aa4-4304-8603-c4de02c4a55f
                         capacity left in the zone requested.</para>
       </note>
     </section>
-    <section>
-      <title>Availability zone aware scheduler</title>
-    </section>
```

Comment:
So this shouldn't be removed, it's just not formatting quite right. This needs to be the section title, and the Network Scheduler needs to be the sub title within that. See: https://docs.openstack.org/newton/networking-guide/config-az.html

Reply:
Ugh hang on, that link is too many versions old.

Reply:
https://docs.openstack.org/neutron/pike/admin/config-az.html#availability-zone-aware-scheduler

Reply:
This one should be better nested